Examples/NoModules/Chapter 15/Ex15_07A/Carton.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CARTON_H
+#define CARTON_H
+#include <algorithm>    // For std::max()
+#include <string>
+#include <string_view>
+#include "Box.h"
+
+class Carton : public Box
+{
+public:
+  // Constructor explicitly calling the base constructor
+  Carton(double l, double w, double h, std::string_view mat = "cardboard")
+    : Box{l, w, h}, m_material{mat}
+  {}
+  ~Carton() override = default;
```

Comment:
I've never seen override on a dtor before.
Of course that works, but do you do that yourself?

Reply:
When I'm adding a destructor, and it overrides, yes. Personally I would not add a defaulted destructor in the header, though. But someone else I know, and who knows what he's doing, does it all the time: his motivation is that this prevents someone from removing the virtual from the base class destructor. Hard to argue with that, because it does make sense...